sources/corepackUtils.ts

```diff
@@ -284,6 +264,30 @@ export async function installVersion(installTarget: string, locator: Locator, {s
   };
 }
 
+async function renameUnderWindows(oldPath: fs.PathLike, newPath: fs.PathLike) {
+  // Windows malicious file analysis blocks files after download so we need to wait for file release
+  const retries = 5;
+  for (let i = 0; i < retries; i++) {
+    try {
+      await fs.promises.rename(oldPath, newPath);
+      break;
+    } catch (err) {
+      if (
+        (
+          (err as nodeUtils.NodeError).code === `ENOENT` ||
+          (err as nodeUtils.NodeError).code === `EPERM`
+        ) &&
+        i < (retries - 1)
+      ) {
+        await new Promise(resolve => setTimeout(resolve, 100));
```

Comment:
Can you use `setTimeout` from `node:timers/promises` please?

Reply:
sure

Reply:
Is it okay like this or would it be better without the alias?